cli/printers/json_printer.py

```diff
@@ -28,3 +31,7 @@ def _get_json_result(self, detections):
         }
 
         return json.dumps(result, indent=4)
+
+    def _exclude_detection_peroperties(self, detections: List[Detection]):
```

Comment:
please don't do it. it will be removed in all the scan types. in addition I don't see any reason to remove data from the response

Reply:
but it's our unique detection id and I amn't sure that user should get it

Reply:
Please remove it,  as discussed

Reply:
Fixed